doc/Changelog.md

```diff
@@ -3,6 +3,15 @@ Changelog {#changelog}
 
 # Release 1.2 (git master)
 
+* [90](https://github.com/BlueBrain/Tide/pull/90):
+  The fullscreen mode has some new features:
+  - Users can now enlarge and pan contents (without the need to use the zoom)
+  - A double-tap toggles between *adjusted* and *maximized* fullscreen
```

Comment:
Ouch, so my descriptions are not clear :-( Maybe we can discuss how to improve that.
There are two fullscreen presets, like a movie player on a phone: one fullscreen without cropping but with black borders, and one *maximized* fullscreen with no black borders but then the content is cropped. Double tap alternates between the two presets, but users can also pinch and pan to adjust differently or zoom in even more (useful for big images). This really only improves the fullscreen mode and implies no changes to the focus mode.